tests/data/formats/SPDXXmlExample.xml

```diff
@@ -67,10 +73,10 @@
 				</packageVerificationCode>
 				<licenseConcluded>(LicenseRef-1 AND MPL-1.1 AND LicenseRef-2 AND LicenseRef-3 AND Apache-2.0 AND LicenseRef-4 AND Apache-1.0)</licenseConcluded>
 				<supplier>Organization: Linux Foundation</supplier>
-				<checksums>
+				<checksum>
```

Comment:
I think the examples in tools-java are a good reference for what the xml structure should look like, e.g. https://github.com/spdx/tools-java/blob/92e1c5d29eacb0081139b0c05e5e6270b231788c/testResources/SPDXXMLExample-v2.3.spdx.xml
Apart from that, I also raised a question on the SPDX tech mailing list (https://lists.spdx.org/g/Spdx-tech) whether there is any reference for the xml schema, because it is necessarily different from e.g. json. Let's see...

Reply:
I have imported the example files from the tools-java and adjusted (at least some) of the test files to use them.